providers/org/datacite/normalizer.py

```diff
@@ -584,3 +600,10 @@ def list_in_string(self, string, list_):
                 if word.search(string):
                     return True
         return False
+
+
+class DataciteNormalizer(Normalizer):
+
+    def do_normalize(self, data):
```

Comment:
@chrisseto, so just call `super`? I need something there so I can import the class right?